### PR TITLE
feat: Adds Tooltip types

### PR DESCRIPTION
**Pull request by Golodhros**
## Description

Adds the tooltip types

NOTE: There is a bug somewhere as the tooltip only shows properly on the first item, but I couldn't figure out what's the problem.

## Motivation and Context
Completing our types

## How Has This Been Tested?
Manually on sandbox

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Build is successful
- [ ] Updated the documentation
- [ ] Added tests to cover changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: src/sandbox/tooltip.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { line, tooltip, LineChartModule } from 'britecharts';
+
+export const constructChart = (containerNode: Element): {lineChartWithTooltip: LineChartModule, tooltipComponent:any} => {
```
Comment (DalerAsrorov) [inline src/sandbox/tooltip.ts:3]: Is any the best usage scenario here?

File: src/typings/charts/tooltip.d.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+>;
+
+export type TooltipOffset = {
+  x: number;
+  y: number;
+};
+
+export type TopicColorMap = Record<string, string>;
+
+type FormattingFunction = (value: number) => number;
```
Comment (DalerAsrorov) [inline src/typings/charts/tooltip.d.ts:38]: Seems like there is no need to create a separate declaration for this, can directly use the (value:number) => number

**Review by DalerAsrorov — CHANGES_REQUESTED** [review]
A few minor comments but looks good otherwise!

File: src/sandbox/tooltip.ts (hunk 1 above)
Comment (Golodhros) [reply src/sandbox/tooltip.ts:3]: True, I will use TooltipModule instead

File: src/typings/charts/tooltip.d.ts (hunk 2 above)
Comment (Golodhros) [reply src/typings/charts/tooltip.d.ts:38]: I think it looks more tidy this way, but next time I will leave it in place

**Review by DalerAsrorov — APPROVED** [review]

**Pull request merged**